diesel/src/query_builder/insert_statement/mod.rs

```diff
@@ -32,6 +32,7 @@ use sqlite::{Sqlite, SqliteConnection};
 /// [`values`]: #method.values
 /// [`default_values`]: #method.default_values
 #[derive(Debug, Clone, Copy)]
+#[must_use="Queries are only executed when calling `load`, `get_result` or similar."]
```

Comment:
I wasn't sure if adding the `must_use` here was necessary or not, feedback welcome